apps/integration/src/app/integrations/modal-wait/modal-wait.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  inject,
+} from '@angular/core';
+import { SkyModalService } from '@skyux/modals';
+
+import { ModalWaitModalComponent } from './modal-wait-modal.component';
+
+@Component({
+  selector: 'app-modal-wait',
+  templateUrl: './modal-wait.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ModalWaitComponent {
+  public isWaiting = false;
+
+  #changeDetector = inject(ChangeDetectorRef);
+  #modalService: SkyModalService;
+
+  constructor(modalService: SkyModalService) {
```

Comment:
Why have this as a constructor parameter instead of using `inject()` like with `ChangeDetectorRef`?

Reply:
🤦 Done